ios/StripeTerminalReactNative.swift

```diff
@@ -423,23 +447,40 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
 
     @objc(collectPaymentMethod:resolver:rejecter:)
     func collectPaymentMethod(params: NSDictionary, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
-
+        
         guard let id = params["paymentIntentId"] as? String else {
             resolve(Errors.createError(code: CommonErrorType.InvalidRequiredParameter, message: "You must provide paymentIntentId."))
             return
```

Comment:
as mentioned in another place we're going to need to figure out an alternative to the paymentIntentId for this to support offline mode. Going to start a slack thread to discuss.